Add FAQ about CoinJoin via tBTC (#177)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

Wasabi is saving on mining fees by setting a confirmation target of roughly 12 hours, but you can re-register unconfirmed anonset outputs for the next round of CoinJoin.
:::

:::details

> ### I'd like to experience CoinJoin but I'm not comfortable with using real Bitcoin. What can I do?

Reviewer: ```suggestion
### I'd like to experience CoinJoin but I'm not comfortable using real Bitcoin. What can I do?
```

> You can try to make a CoinJoin via Wasabi by using tBTC (testnet BTC).

Reviewer: ```suggestion
You can try to make a CoinJoin via Wasabi on the Bitcoin TestNet (an alternative Bitcoin blockchain, to be used for testing).
Go to settings and change the network to TestNet.
```

Just download Wasabi and and get free tBTC from faucets like:

> [Coinfaucet.eu](https://coinfaucet.eu/en/btc-testnet/)

Reviewer: ```suggestion
[https://coinfaucet.eu/en/btc-testnet/](https://coinfaucet.eu/en/btc-testnet/)
```

or

> [Mempool.co](https://testnet-faucet.mempool.co/)

Reviewer: ```suggestion
[https://testnet-faucet.mempool.co/](https://testnet-faucet.mempool.co/)
```

Wasabi is saving on mining fees by setting a confirmation target of roughly 12 hours, but you can re-register unconfirmed anonset outputs for the next round of CoinJoin.
:::

:::details
### I'd like to experience CoinJoin but I'm not comfortable using real Bitcoin. What can I do?

You can try to make a CoinJoin via Wasabi on the Bitcoin TestNet (an alternative Bitcoin blockchain, to be used for testing).
Go to settings and change the network to TestNet.

> Just download Wasabi and and get free tBTC from faucets like:

Reviewer: It doesn't make sense to say `Just download Wasabi` after saying `Go to settings ...`
```suggestion
You can get tBTC from faucets like:
```
